pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -205,3 +209,13 @@ func ManagedTeamRoleName(teamID int64) string {
 func ManagedBuiltInRoleName(builtInRole string) string {
 	return fmt.Sprintf("managed:builtins:%s:permissions", strings.ToLower(builtInRole))
 }
+
+func extractPrefixes(prefix string) (string, string, bool) {
+	parts := strings.Split(strings.TrimSuffix(prefix, ":"), ":")
+	if len(parts) != 2 {
```

Comment:
though: as @gamab pointed out, this will not work for our `global:users:...` scope. I don't think we're returning metadata for it, so it's not currently a problem. But, as we're now coding in assumptions about the number of scope parts, we should probably document scope best practices :)

Reply:
Yeah, I like that. In general, we should clean up scopes and document the expected scope structure.